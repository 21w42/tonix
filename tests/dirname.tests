dirname t
dirname --help
help dirname
man dirname
dirname /usr/share
dirname /etc/motd
quit

