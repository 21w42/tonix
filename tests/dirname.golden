Welcome to Tonix.
Type "help" to get a list of commands.
"man <COMMAND>" or "help <COMMAND>" sometimes might be helpful.
Some options for certain commands work as well.
Feel free to navigate a pre-made file system using intuitive commands.
Path resolution does not work yet, one step at a time please.
Your feedback is highly appreciated!
Have fun :)
$ t

$ Usage: 	dirname [OPTION] NAME...

Output NAME with its last non-slash component and trailing slashes removed; if NAME contains no /'s, output '.' (meaning the current directory).
	-z		end each output line with NUL, not newline
	--help		display this help and exit
	--version	output version information and exit

$ Usage: 	dirname [OPTION] NAME...

Output NAME with its last non-slash component and trailing slashes removed; if NAME contains no /'s, output '.' (meaning the current directory).
	-z		end each output line with NUL, not newline
	--help		display this help and exit
	--version	output version information and exit

$ dirname		User Commands
NAME
	dirname - strip last component from file name
SYNOPSIS
	dirname [OPTION] NAME...
DESCRIPTION
	Output NAME with its last non-slash component and trailing slashes removed; if NAME contains no /'s, output '.' (meaning the current directory).
	-z		end each output line with NUL, not newline
	--help		display this help and exit
	--version	output version information and exit

$ /usr

$ /etc

$ Bye.
